lib/data/recruitment_banners.yml

```diff
@@ -16,3 +16,27 @@ banners:
   page_paths:
     - /guidance/keeping-your-hmrc-login-details-safe
     - /government/collections/hmrc-phishing-and-scams-detailed-information
+- name: AI banner 11/11/2024
+  suggestion_text: "Help improve GOV.UK"
+  suggestion_link_text: "Sign up to take part in user research (opens in a new tab)"
+  survey_url: https://survey.take-part-in-research.service.gov.uk/jfe/form/SV_2bggmg6xlelrO0S
+  page_paths:
+    - /self-assessment-tax-returns
+    - /working-for-yourself
+    - /self-employed-records
+    - /expenses-if-youre-self-employed
+    - /first-company-accounts-and-return
+    - /what-is-the-construction-industry-scheme
+    - /capital-allowances
+    - /simpler-income-tax-cash-basis
+    - /expenses-and-benefits-a-to-z
+    - /capital-gains-tax
+    - /directors-loans
+    - /self-assessment-tax-return-forms
+    - /running-a-limited-company
+    - /calculate-tax-on-company-cars
+    - /introduction-to-business-rates
+    - /calculate-your-business-rates
+    - /apply-for-business-rate-relief
+    - /stop-being-self-employed
+    - /tax-codes
```

Comment:
```suggestion
    - /tax-codes
    
```

Reply:
I can't spot the issue here??

Reply:
It's not a blocker. The file is missing the new line at the end.

Reply:
Ah ok! Thank you!